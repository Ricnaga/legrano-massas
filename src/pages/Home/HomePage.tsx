import { Fade, Grid, Grow } from '@mui/material';
import logo from '../../assets/logo.png';
import { HomeContacts } from './components/HomeContacts';
import { StyledAvatar, StyledGrid } from './HomePage.styled';

export function HomePage() {
  return (
    <StyledGrid container>
      <Grid item xs={12} container justifyContent="center" marginY={4}>
        <Grow in>
          <StyledAvatar src={logo} />
        </Grow>
      </Grid>
      <Fade in>
        <Grid item xs={12} container justifyContent="center">
          <HomeContacts />
        </Grid>
      </Fade>
    </StyledGrid>
  );
}
