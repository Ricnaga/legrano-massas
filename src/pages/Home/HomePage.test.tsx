import React from 'react';
import { renderWithTheme } from '../../application/test/testing-library';
import { HomePage } from './HomePage';

describe('HOME SCREEN', () => {
  it('should render correctly', () => {
    const { container } = renderWithTheme(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
