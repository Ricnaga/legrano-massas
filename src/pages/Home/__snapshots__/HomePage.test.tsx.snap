// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: HomePage should render correctly 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container css-1hmj4x9-MuiGrid-root"
  >
    <div
      class="MuiGrid-root css-1y66wer-MuiGrid-root"
    >
      <img
        alt="Logo do Legrano"
        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root"
        src="logo.png"
        style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
    </div>
    <div
      class="MuiGrid-root css-vj1n65-MuiGrid-root"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiGrid-root css-1126moi-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
            data-testid="WhatsAppIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.75 13.96c.25.13.41.2.46.3.06.11.04.61-.21 1.18-.2.56-1.24 1.1-1.7 1.12-.46.02-.47.36-2.96-.73-2.49-1.09-3.99-3.75-4.11-3.92-.12-.17-.96-1.38-.92-2.61.05-1.22.69-1.8.95-2.04.24-.26.51-.29.68-.26h.47c.15 0 .36-.06.55.45l.69 1.87c.06.13.1.28.01.44l-.27.41-.39.42c-.12.12-.26.25-.12.5.12.26.62 1.09 1.32 1.78.91.88 1.71 1.17 1.95 1.3.24.14.39.12.54-.04l.81-.94c.19-.25.35-.19.58-.11l1.67.88M12 2a10 10 0 0 1 10 10 10 10 0 0 1-10 10c-1.97 0-3.8-.57-5.35-1.55L2 22l1.55-4.65A9.969 9.969 0 0 1 2 12 10 10 0 0 1 12 2m0 2a8 8 0 0 0-8 8c0 1.72.54 3.31 1.46 4.61L4.5 19.5l2.89-.96A7.95 7.95 0 0 0 12 20a8 8 0 0 0 8-8 8 8 0 0 0-8-8z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-b240w0-MuiTypography-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1vcnyyg-MuiTypography-root-MuiLink-root"
              href="https://wa.me/5511910371645"
            >
              (11) 91037-1645
            </a>
          </p>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
            data-testid="InstagramIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-b240w0-MuiTypography-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1vcnyyg-MuiTypography-root-MuiLink-root"
              href="https://www.instagram.com/legrano_"
            >
              @legrano_
            </a>
          </p>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
            data-testid="FacebookIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-b240w0-MuiTypography-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1vcnyyg-MuiTypography-root-MuiLink-root"
              href="https://pt-br.facebook.com/legrano.culinariaartesanal"
            >
              legrano.culinariaartesanal
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
